opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
```

Comment:
@destogl about moving the code _to_variant:

No problem I will make the change, only I find it not very clear code that the datatype is extract in something called _to_variant.

In the current and new version you still can you use a variant as value and also use the datatype, only as long as you set vtype to None. Looks error prone ...

side mark; personally I find the construction with multiple ( use of *args a little bit vague, makes it harder to understand what the exact signature of a method is.